kustomize/commands/edit/add/addmetadata.go

```diff
@@ -112,6 +116,9 @@ func (o *addMetadataOptions) validateAndParse(args []string) error {
 	if len(args) < 1 {
 		return fmt.Errorf("must specify %s", o.kind)
 	}
+	if o.labelsWithoutSelector == false && o.includeTemplates == true {
```

Comment:
nit: you could write like this for brevity:
```suggestion
	if !o.labelsWithoutSelector && o.includeTemplates {
```
since both flags are boolean